src/modules/core/utils/array.utils.ts

```diff
@@ -0,0 +1,3 @@
+export const range = (start: number, length: number): number[] => {
```

Comment:
I suggest putting length as the first parameter and making start optional with default to 0. And adding JSDoc.